.github/CODEOWNERS

```diff
@@ -283,6 +283,7 @@
 /pkg/services/sqlstore/migrations/ualert/ @grafana/alerting-backend-product
 /pkg/tests/api/alerting/ @grafana/alerting-backend-product
 /public/app/features/alerting/ @grafana/alerting-frontend
+/public/app/features/gops/ @grafana/alerting-frontend
```

Comment:
I was not sure about who should own these files, maybe we could add the rest of frontend teams?

Reply:
I think it's fine if we owned it :)